src/main/java/controllers/user/dao/UserDAO.java

```diff
@@ -46,36 +46,57 @@ public static UserAPI createUser(final UserAPI userToCreate) throws SQLException
 	{
 		// Convert
 		final UserDB dbObject = UserFactory.APIToDB(userToCreate);
-		
+
 		// Create query
 		final String query = String.format("INSERT INTO %s (school_id) VALUES('%s') RETURNING *",
 				Table.USER.TableName, dbObject.schoolId);
-		
+
 		// Execute
-		try { return queryDatabase(query)[0]; }
-		catch (Exception e) {
-			System.out.println("UserDAO.createUser() failed:\n" + e.getMessage()); throw e;
+		try {
+			return queryDatabase(query)[0];
+		} catch (Exception e) {
+			System.out.println("UserDAO.createUser() failed:\n" + e.getMessage());
+			throw e;
 		}
 	}
-	
+
+	/**
+	 * @param schoolId is "schoolId" or external ID
+	 * @return the queried user
+	 * @throws SQLException if operation fails
+	 * @author Erlend
+	 */
+	public static UserAPI[] retrieveUser(String schoolId) throws SQLException {
+		String schoolIdClean = schoolId.replaceAll("[^a-zA-Z0-9æøå]", "");
```

Comment:
Hvad er det du gør her?

Reply:
Jeg stopper eventuelle forbudte tegn fra DTU. Jeg kan godt se, den burde kaste en exception. Den er ikke essentiel for funktionen.